vocab-builder-core/src/main/java/com/github/tkurz/sesame/vocab/VocabBuilder.java

```diff
@@ -183,11 +184,41 @@ public void generate(String className, PrintWriter out) throws IOException, Grap
         out.printf(getIndent(1) + "public static final String PREFIX = \"%s\";%n", name.toLowerCase());
         out.println();
 
-        //and now the resources
         List<String> keys = new ArrayList<>();
         keys.addAll(splitUris.keySet());
         Collections.sort(keys, String.CASE_INSENSITIVE_ORDER);
 
+        //string constant values
+        if (StringUtils.isNotBlank(stringPropertyPrefix) || (StringUtils.isNotBlank(stringPropertySuffix))) {
+            // add the possibility to add a string property with the namespace for usage in
+            for (String key : keys) {
+                Literal comment = getFirstExistingObjectLiteral(model, splitUris.get(key), getPreferredLanguage(), COMMENT_PROPERTIES);
+                Literal label = getFirstExistingObjectLiteral(model, splitUris.get(key), getPreferredLanguage(), LABEL_PROPERTIES);
+
+                out.println(getIndent(1) + "/**");
+                if (label != null) {
+                    out.printf(getIndent(1) + " * %s%n", label.getLabel());
+                    out.println(getIndent(1) + " * <p>");
+                }
+                out.printf(getIndent(1) + " * {@code %s}.%n", splitUris.get(key).stringValue());
+                if (comment != null) {
+                    out.println(getIndent(1) + " * <p>");
+                    out.printf(getIndent(1) + " * %s%n", WordUtils.wrap(comment.getLabel().replaceAll("\\s+", " "), 70, "\n" + getIndent(1) + " * ", false));
+                }
+                out.println(getIndent(1) + " *");
+                out.printf(getIndent(1) + " * @see <a href=\"%s\">%s</a>%n", splitUris.get(key), key);
+                out.println(getIndent(1) + " */");
+
+                String nextKey = cleanKey(String.format("%s%s%s", StringUtils.defaultString(stringPropertyPrefix),
+                        doCaseFormatting(key, CaseFormat.UPPER_UNDERSCORE),
```

Comment:
For pretty code ;-)

Personally, I prefer having the URI constants unchanged, the String constants in UPPER_UNDERSCORE. But I see the point, I'll add a parameter to configure this.